source/_integrations/zwave_js.markdown

```diff
@@ -57,7 +57,7 @@ To run a Z-Wave network, you need the following elements:
 
 ### Setting up a Z-Wave JS server
 
-The easiest way to get started is by using the built-in Z-Wave JS add-on in Home Assistant.
+If you are running Home Assistant OS or Supervisor, the easiest way to get started is by using the built-in Z-Wave JS add-on in Home Assistant.
```

Comment:
- The installation method is called "Home Assistant Operating System", not Home Assistant OS.
- The Supervisor will run on OS and cannot be run separately, I think you meant the "Home Assistant Supervised" installation method, and not the Supervised application.

../Frenck

Reply:
@frenck I thought about messaging you directly, but because I find it important to make a more generic point, I'm replying here. I know how open source works. I know about pull requests, branches, commits and so on - they're part of my day-to-day work, although not a big part since I'm not a full-time software developer.

In this case, all I tried was to make the doc just that more readable, based on this issue: https://github.com/home-assistant/home-assistant.io/issues/28062. In a comment on this, @MartinHjelmare stated that the doc "assumes you're running Home Assistant OS or Supervisor which can install add-ons". I used those exact terms. And I'm not the only one on this exact doc page: under the Advanced installation instructions, the term Home Assistant OS is also used.

It is a small clarification. Not a code change, just a remark in the documentation to move away from assumptions - just because many people know what assumptions are: the mother of all... foul-ups. In this case, an assumption that cost me a few hours of trying to get something to work, only to find out it never will.

The attitude towards contributions in general must be one of seeing the value. Not one of nitpicking on semantics in a minor doc contribution. Your review is the complete opposite of that - something I see in many, many contributions both here on Github and in the various forums. I have seen quite a few people mentioning that because of the reactions they get, they now run other platforms. And that is my generic point: as far as I'm concerned, your comments are an example of how not to treat a contribution.

Reply:
> Your review is the complete opposite of that 

Sorry, but you are entirely pulling this out of context, and I don't understand why you see a review as something negative. See my comment below here for earlier, which fully applies as a response to your message here:

<https://github.com/home-assistant/home-assistant.io/pull/28063#issuecomment-1619966934>

I'm was here to help, not to punch you in the face.

> And that is my generic point: as far as I'm concerned, your comments are an example of how not to treat a contribution.

That comment makes no sense whatsoever, are you saying we should just accept and merge everything, regardless if it is correct or not? 🤷 

../Frenck

Reply:
I think he was just making a correction in line with our documentation standards: https://developers.home-assistant.io/docs/documenting/standards#general-documentation

> The case of brand names, services, protocols, integrations and platforms must match its respective counterpart. E.g., "Z-Wave" not "Zwave", "Z-wave", "Z Wave" or "ZWave". Also, "Input Select" not "input select" or "Input select".

I'd also suggest you read this section in response to getting feedback on PRs: https://developers.home-assistant.io/docs/review-process#receiving-review-comments

> Be very aware these review comments are not personal. The reviewer is not trying to insult you or make you feel bad. They are trying to help you improve your PR, so it can be merged.

While PR comments/corrections can come across as negative, he was just being direct. Frenck reviews a lot of PRs and I can see it becoming exhausting if he tried to sugar coat everything.

Reply:
> but it is by no means incorrect. 

It is. Sorry, hence my comment. Those refer to application parts, not installation methods. If you think we should have reference application parts, the suggestion in this PR would have been a duplicate, as the OS cannot function without the Supervisor application.

> However, you apparently think it's necessary to use your power to not approve the PR

As a reviewer, like any other reviewer, it is my task to verify the PR, and yes, just like any other reviewer, I will ask for changes in case we've seen an issue with it. In this case, I've provide a small suggestion in the wording used.

>  and to treat me as a little schoolboy by telling me to go back and make the wording exactly as you want it.

I did no such thing; I left a review comment pointing out something was not fully correct in your suggestion here: <https://github.com/home-assistant/home-assistant.io/pull/28063#discussion_r1251803317>, and even provided the suggested wording to use instead.

> is that my work isn't good enough in the eyes of the Lord Frenck Almighty

That is correct, it wasn't correct. You seem not be able to see that?

Anyways, that comment is close to getting personal and mocking behavior. Please don't go that route, we have a code of conduct that disallow such behavior in our community, which is also applies to our GitHub repositories.

>  That is what started all of this and it is on you.

Sorry, it isn't. You don't have to agree; that is fine. But there was nothing out of the ordinary or wrong with this code review. Your response, however, has been slightly out of the ordinary.

Your behavior is noticed in the community as well, as it was discussed in our Discord chats by others. From that perspective, it was useful, as another person stood up and actually did make the suggested correction, which has been merged and published.

Although I really rather have merged the original PR after a tweak was made.

../Frenck

Reply:
@gleversluis it took me less than 5 minutes to change "Home Assistant OS" to "Home Assistant Operating System" and create a new PR, submit it, and merge it.  You've spent more time chastising others than it would have taken to fix the PR.  Your response was so outlandish, that it took notice to many others in the community.  Please leave your ego at the door in the future when creating PRs.  Reviews and comments are quite normal during the PR creation process.  Getting overly upset about a review or comment is not normal.